tests/parsing/errors/other/expected/spread.res.txt

```diff
@@ -42,79 +42,64 @@ Explanation: since records have a known, fixed shape, a spread like `{a, ...b}`
   4 │ let record = {...x, ...y}
   5 │ let {...x, ...y} = myRecord
   6 │ 
-  7 │ let myList = list{...x, ...y}
+  7 │ let list{...x, ...y} = myList
 
   Record's `...` spread is not supported in pattern matches.
 Explanation: you can't collect a subset of a record's field into its own record, since a record needs an explicit declaration and that subset wouldn't have one.
 Solution: you need to pull out each field you want explicitly.
 
 
   Syntax error!
-  tests/parsing/errors/other/spread.res:8:1-3
+  tests/parsing/errors/other/spread.res:7:13-22
 
-   6 │ 
-   7 │ let myList = list{...x, ...y}
-   8 │ let list{...x, ...y} = myList
-   9 │ 
-  10 │ type t = {...a}
-
-  Lists can only have one `...` spread, and at the end.
-Explanation: lists are singly-linked list, where a node contains a value and points to the next node. `list{a, ...bc}` efficiently creates a new item and links `bc` as its next nodes. `list{...bc, a}` would be expensive, as it'd need to traverse `bc` and prepend each item to `a` one by one. We therefore disallow such syntax sugar.
-Solution: directly use `concat`.
-
-
-  Syntax error!
-  tests/parsing/errors/other/spread.res:8:13-22
-
-   6 │ 
-   7 │ let myList = list{...x, ...y}
-   8 │ let list{...x, ...y} = myList
-   9 │ 
-  10 │ type t = {...a}
+  5 │ let {...x, ...y} = myRecord
+  6 │ 
+  7 │ let list{...x, ...y} = myList
+  8 │ 
+  9 │ type t = {...a}
 
   List pattern matches only supports one `...` spread, at the end.
 Explanation: a list spread at the tail is efficient, but a spread in the middle would create new lists; out of performance concern, our pattern matching currently guarantees to never create new intermediate data.
```

Comment:
we may need update the docs about pattern in later commits, it is not supported since it is not deterministic